connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -680,18 +680,24 @@ abstract class AvroSuite
       val zstandardDir = s"$dir/zstandard"
 
       val df = spark.read.format("avro").load(testAvro)
-      spark.conf.set(SQLConf.AVRO_COMPRESSION_CODEC.key, "uncompressed")
+      spark.conf.set(SQLConf.AVRO_COMPRESSION_CODEC.key,
+        AvroCompressionCodec.UNCOMPRESSED.name().toLowerCase(Locale.ROOT))
```

Comment:
How about adding a `lowerCaseName()` method to `AvroCompressionCodec`?

Reply:
I have considered doing this before. But these usage just exists in test cases. So I think it's not worth.
If the non test path need, we add it at that time.